contracts/src/deployers/aerodrome-lp/AerodromeLpTarget0Deployer.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.22;
+
+import { IGauge } from "aerodrome/interfaces/IGauge.sol";
+import { AerodromeLpTarget0 } from "../../instances/aerodrome-lp/AerodromeLpTarget0.sol";
+import { IHyperdrive } from "../../interfaces/IHyperdrive.sol";
+import { IHyperdriveAdminController } from "../../interfaces/IHyperdriveAdminController.sol";
+import { IHyperdriveTargetDeployer } from "../../interfaces/IHyperdriveTargetDeployer.sol";
+
+/// @author DELV
+/// @title AerodromeLpTarget0Deployer
+/// @notice The target0 deployer for the AerodromeLpHyperdrive implementation.
+/// @custom:disclaimer The language used in this code is for coding convenience
+///                    only, and is not intended to, and does not, have any
+///                    particular legal or regulatory significance.
+contract AerodromeLpTarget0Deployer is IHyperdriveTargetDeployer {
+    /// @dev The Aerodrome Gauage contract. This is where the base token will be
+    ///      deposited.
+    IGauge internal immutable gauge;
+
+    /// @notice Instantiates the AerodomeLpHyperdrive base contract.
+    /// @param _gauge The Aerodrome Gauage contract.
+    constructor(IGauge _gauge) {
+        gauge = _gauge;
+    }
+
```

Comment:
We'll also need to remove this from the other deployer contracts. This will need to be passed as extra data.